vochain/transaction.go

```diff
@@ -97,6 +98,14 @@ func AddTx(vtx *models.Tx, state *State, txID [32]byte, commit bool) ([]byte, er
 					return []byte{}, fmt.Errorf("set process results, results is nil")
 				}
 				return []byte{}, state.SetProcessResults(tx.ProcessId, tx.Results, true)
+			case models.TxType_SET_PROCESS_CENSUS:
+				if bytes.Equal(tx.CensusRoot, []byte{}) {
+					return []byte{}, fmt.Errorf("set process census, census root is empty")
+				}
+				if *tx.CensusURI == "" {
```

Comment:
That's a PANIC if tx.CensusURI == nil since tx.CensusURI is a pointer.

When dealing with the Protobuf models that's something very critic, any pointer field must be checked before accessing to it. Please, make sure all proto accesses are safe.